src/bundle/Controller/SearchController.php

```diff
@@ -175,8 +175,9 @@ public function searchAction(Request $request): Response
                 if (!empty($criteria)) {
                     $query->filter = new Criterion\LogicalAnd($criteria);
                 }
-
-                $query->sortClauses[] = new SortClause\DateModified(Query::SORT_ASC);
```

Comment:
It might be cleaner to check for scoring capability, if not present, the sorting by date can be a fallback.
ref: https://github.com/ezsystems/ezplatform-solr-search-engine/blob/master/lib/Handler.php#L451

Reply:
Good catch, I wasn't aware that this is possible to check whether `SearchService` supports some features. Thanks!

Reply:
Good catch, I wasn't aware that this is possible to check whether SearchService supports some features. Thanks!